contributors/devel/api-conventions.md

```diff
@@ -1240,6 +1240,8 @@ latter if the label/annotation is specific to `foo`
 the resource doesn't need to know about, experimental fields that aren't
 intended to be generally used API fields, etc. Beware that annotations aren't
 automatically handled by the API conversion machinery.
+  - The key name of [taint](https://kubernetes.io/docs/concepts/configuration/taint-and-toleration/) should also follow above naming conversion.
+    - For instance, prefer taint `foo.kubernetes.io/foo-bar` over `foo.kubernetes.io/fooBar`
```

Comment:
Recommend rewording this to just "Use dashes instead of camelCase. For instance, prefer `foo.kubernetes.io/foo-bar` over `foo.kubernetes.io/fooBar`"